### PR TITLE
ISSUE-560 Change email addresses contacts

### PR DESCRIPTION
**Pull request by Arsnael**
The code base is ready. I just need to add documentation for it :)

### CODE REVIEW

File: tmail-backend/integration-tests/webadmin/webadmin-integration-tests-common/src/main/java/com/linagora/tmail/integration/UsernameChangeIntegrationContract.java [hunk 1]
```diff
@@ -32,6 +32,8 @@ public class UsernameChangeIntegrationContract {
 
     private RequestSpecification webAdminApi;
 
+    public abstract void awaitDocumentsIndexed(Long documentCount);
```
[the remark this answers is not in the document]
Comment (quantranhong1999) [reply tmail-backend/integration-tests/webadmin/webadmin-integration-tests-common/src/main/java/com/linagora/tmail/integration/UsernameChangeIntegrationContract.java:35]: BTW 
```java

[ERROR]   DistributedUsernameChangeIntegrationTest>UsernameChangeIntegrationContract.shouldAdaptContacts:69 

Expecting ArrayList:

  []

to contain only:

  [EmailAddressContact(a261f484-d0ff-33af-8a33-7b474b26a206,ContactFields(andre@linagora.com,Andre,Dupont))]

but could not find the following element(s):

  [EmailAddressContact(a261f484-d0ff-33af-8a33-7b474b26a206,ContactFields(andre@linagora.com,Andre,Dupont))]
```

**Review by quantranhong1999 — APPROVED** [review]
Lgtm

File: tmail-backend/integration-tests/webadmin/webadmin-integration-tests-common/src/main/java/com/linagora/tmail/integration/UsernameChangeIntegrationContract.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Arsnael) [reply tmail-backend/integration-tests/webadmin/webadmin-integration-tests-common/src/main/java/com/linagora/tmail/integration/UsernameChangeIntegrationContract.java:35]: Ah good point I forgot to use it in the test... Somehow in my local it passes so I didnt notice it^^'

**Comment by Arsnael** [comment]
Still having the same issue... I will need to investigate deeper when I have time

File: tmail-backend/integration-tests/webadmin/webadmin-integration-tests-common/src/main/java/com/linagora/tmail/integration/UsernameChangeIntegrationContract.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            .get("taskId");
+
+        webAdminApi.get("/tasks/" + taskId + "/await");
+
+        awaitDocumentsIndexed(1L);
+
+        assertThat(contactProbe.list(BOB_ACCOUNT_ID))
+            .containsOnly(contact);
+        assertThat(contactProbe.list(ALICE_ACCOUNT_ID))
+            .isEmpty();
```
[the remark this answers is not in the document]
Comment (Arsnael) [reply tmail-backend/integration-tests/webadmin/webadmin-integration-tests-common/src/main/java/com/linagora/tmail/integration/UsernameChangeIntegrationContract.java:75]: Nice thinking :) I will try that

**Review by quantranhong1999 — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Pull request merged**